src/iqm/benchmarks/benchmark_definition.py

```diff
@@ -34,6 +34,52 @@
 from iqm.qiskit_iqm.iqm_provider import IQMBackend, IQMFacadeBackend
 
 
+@dataclass
+class ObservationIdentifier:
+    """Identifier for observations for ease of use
+
+    Attributes:
+        qubit_indices: list containing the indices of the qubits the observation was executed on.
+    """
+
+    qubit_indices: list[int]
+
+    @property
+    def string_identifier(self) -> str:
+        """String version of the qubit indices for ease of use
+
+        Returns:
+            A string of the qubit indices
+        """
+        return str(self.qubit_indices)
+
+
+@dataclass
+class Observation:
+    """Dataclass to store the main results of a single run of a Benchmark
+
+    Attributes:
+        name: name of the observation
+        value: value of the observation
+        identifier: identifier, which should be a string of the qubit layout
+        uncertainty: uncertainty of the observation
+    """
+
+    name: str
+    value: Any
+    identifier: ObservationIdentifier
+    uncertainty: Optional[Any] = None
+
+    # @property
```

Comment:
commented out — intentionally kept here?

Reply:
That is forgotten, I'll remove and push changes